app/Http/Middleware/CheckAdministratorPrivilege.php

```diff
@@ -17,7 +17,7 @@ class CheckAdministratorPrivilege
      */
     public function handle(Request $request, Closure $next)
     {
-        abort_if(! Auth::user()->is_account_administrator, 401);
+        abort_if(! Auth::user()->is_account_administrator, 404);
```

Comment:
Except for this one. We should keep a `401` here

Reply:
done